lte/gateway/c/oai/tasks/mme_app/mme_app_location.c

```diff
@@ -217,7 +218,13 @@ int mme_app_handle_s6a_update_location_ans(
   struct ue_mm_context_s* ue_mm_context = NULL;
   int rc = RETURNok;
 
-  DevAssert(ula_pP);
+  if (ula_pP == NULL) {
+    OAILOG_ERROR(
+      LOG_MME_APP,
+      "Invalid S6a Update Location Answer ITTI message "
```

Comment:
This message string can fit in one line.

Reply:
Fixed in the updated commit.